SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialHandoverManager.cpp

```diff
@@ -0,0 +1,170 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "EngineClasses/SpatialHandoverManager.h"
+
+#include "EngineClasses/SpatialPackageMapClient.h"
+#include "Interop/Connection/SpatialOSWorkerInterface.h"
+#include "SpatialCommonTypes.h"
+#include "SpatialConstants.h"
+#include "SpatialView/EntityDelta.h"
+#include "SpatialView/SubView.h"
+#include "SpatialView/ViewDelta.h"
+
+namespace SpatialGDK
+{
+SpatialHandoverManager::SpatialHandoverManager(const FSubView& InActorView, const FSubView& InPartitionView)
+	: ActorView(InActorView)
+	, PartitionView(InPartitionView)
+{
+}
+
+void SpatialHandoverManager::Advance()
+{
+	for (const EntityDelta& Delta : PartitionView.GetViewDelta().EntityDeltas)
+	{
+		switch (Delta.Type)
+		{
+		case EntityDelta::ADD:
+			PartitionsToACK.Add(Delta.EntityId);
+			OwnedPartitions.Add(Delta.EntityId);
+			break;
+		case EntityDelta::REMOVE:
+			OwnedPartitions.Remove(Delta.EntityId);
+			break;
+		case EntityDelta::TEMPORARILY_REMOVED:
+
```

Comment:
I just don't know if it's needed for now, and I'd rather not forget about it.

Reply:
There are two cases that I think are going to come up in load balancing is where we handle refreshes.

In an auth subview if we ack stuff.
In a non-auth subview where we read data.

If we read data then data might have changed on a refresh so that one isn't too interesting.
If we ack stuff then we might have to know when the thing that we ack'd has finished. A temp remove means that probably happened.